config/build_nrel_kestrel.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#####################################################################################
+## * This script builds available configurations of QMCPACK                        ##
+##   on kestrel at NREL                                                            ##
+##                                                                                 ##
+## * Execute this script in qmcpack git top level directory                        ##
+##   ./config/build_nrel_kestrel.sh                                                ##
+##                                                                                 ##
+## Last verified: July 23, 2024                                                    ##
+#####################################################################################
+
+# module files resulting from module imports below:
+# Currently Loaded Modules:
+#  1) intel-oneapi-compilers/2023.2.0      2) intel-oneapi-mpi/2021.11.0-intel
+#  3) gcc/13.1.0                           4) intel-oneapi-tbb/2021.10.0-intel
+#  5) intel-oneapi-mkl/2023.2.0-intel      6) boost/1.84.0-intel-oneapi-mpi-intel
+#  7) fftw/3.3.10-intel-oneapi-mpi-intel   8) hdf5/1.14.3-intel-oneapi-mpi-intel
+#  9) curl/8.6.0                          10) cmake/3.27.9
+
+source $MODULESHOME/init/bash
+module purge
+module load intel-oneapi-compilers/2023.2.0
+module load intel-oneapi-mpi/2021.11.0-intel
+module load gcc/13.1.0
+module load intel-oneapi-mkl/2023.2.0-intel
+module load boost/1.84.0-intel-oneapi-mpi-intel
+module load fftw/3.3.10-intel-oneapi-mpi-intel
+module load hdf5/1.14.3-intel-oneapi-mpi-intel
+module load cmake/3.27.9
+ 
+module list
+
+CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx\
+             -DENABLE_TIMERS=1 -DBUILD_AFQMC=0"
```

Comment:
`-DENABLE_TIMERS=1 -DBUILD_AFQMC=0` not needed as they are default.

Reply:
Addressed